webapp/fmprod/src/main/java/org/apache/oodt/cas/product/jaxrs/resources/ProductPageResource.java

```diff
@@ -17,143 +17,148 @@
+            List<Reference> references = productReferencesList.get(i);
+
+            this.productResources.add(new ProductResource(product,metadata,productReferencesList.get(0),workingDir));
+        }
+
+
     }
-  }
-
-  /**
-   * Adds a {@link ProductResource} to the list of product resources for the dataset.
-   *
-   * @param resource the resource to add to the dataset.
-   */
-  public void addProductResource(ProductResource resource) {
-    productResources.add(resource);
-  }
-
-  /**
-   * Gets the working directory for the dataset.
-   *
-   * @return the working directory
-   */
-  public File getWorkingDir() {
-    return workingDir;
-  }
-
-  @XmlElement(name = "pageSize")
-  public int getPageSize() {
-    return this.pageSize;
-  }
-
-  @XmlElement(name = "pageNum")
-  public int getPageNum() {
-    return pageNum;
-  }
-
-  @XmlElement(name = "totalPages")
-  public int getTotalPages() {
-    return totalPages;
-  }
-
-  @XmlElement(name = "numOfHits")
-  public long getNumOfHits() {
-    return numOfHits;
-  }
-
-  /**
-   * Gets the product resources for the dataset.
-   *
-   * @return the productResources
-   */
-  @XmlElementWrapper(name = "products")
-  @XmlElement(name = "product")
-  public List<ProductResource> getProductResources() {
-    return productResources;
-  }
-
-  /**
-   * Gets the metadata resource for the dataset.
-   *
-   * @return the metadata resource
-   */
-  @XmlElement(name = "metadata")
-  public MetadataResource getMetadataResource() {
-    return metadataResource;
-  }
+
+
+
+
+
+    /**
+     * Adds a {@link ProductResource} to the list of product resources for the
+     * dataset.
+     * @param resource the resource to add to the dataset.
+     */
+    public void addProductResource(ProductResource resource)
+    {
+        productResources.add(resource);
+    }
+
+
+
+    /**
+     * Gets the working directory for the dataset.
+     * @return the working directory
+     */
+    public File getWorkingDir()
+    {
+        return workingDir;
+    }
+
+
+
+    @XmlElement(name="pageSize")
+    public int getPageSize(){
+        return this.pageSize;
+    }
+
+    @XmlElement(name="pageNum")
+    public int getPageNum() {
+        return pageNum;
+    }
+
+    @XmlElement(name="totalPages")
+    public int getTotalPages() {
+        return totalPages;
+    }
+
+    @XmlElement(name="numOfHits")
+    public long getNumOfHits() {
+        return numOfHits;
+    }
+
+
+    /**
+     * Gets the product resources for the dataset.
+     * @return the productResources
+     */
+    @XmlElementWrapper(name = "products")
+    @XmlElement(name = "product")
+    public List<ProductResource> getProductResources()
+    {
+        return productResources;
+    }
+
+
+
+    /**
+     * Gets the metadata resource for the dataset.
+     * @return the metadata resource
+     */
+    @XmlElement(name = "metadata")
+    public MetadataResource getMetadataResource()
+    {
+        return metadataResource;
+    }
+
```

Comment:
Seems like the file is not formatted correctly (inconsistent spacing between methods and etc). I assume you are using the google code formatter, as we did in a previous PR. Can you check?